mkdocs.yml

```
@@ -140,6 +141,7 @@ nav:
   - 'Учебник на русском языке':       'Tutorial-ru.md'
   - 'Türkçe Kılavuz':                 'Tutorial-tr.md'
   - '简体中文教程':                     'Tutorial-zh.md'
+  - 'Vodnik po uporabi knjižnice':    'Tutorial-sl.md
```

```suggestion
  - 'Vodnik po uporabi knjižnice':    'Tutorial-sl.md'
```
There is a missing single quote 😅